### PR TITLE
[Docs] Copy edit Index, Architecture, Getting Started.

### PR DESCRIPTION
**Pull request by kplybon**

### CODE REVIEW

File: docs/clusterfuzz/architecture.md [hunk 1]
```diff
@@ -21,35 +22,41 @@ triaging crashes, minimizing reproducers, [bisecting], and verification of fixes
 ClusterFuzz is written in Python and Go. It runs on **Linux**, **macOS**, and **Windows**.
 
 ## Requirements
-It runs on the [Google Cloud Platform](https://cloud.google.com/), and depends
+ClusterFuzz runs on the [Google Cloud Platform](https://cloud.google.com/) and depends
 on a number of services:
-- Compute Engine (Not strictly necessary. Bots can run anywhere).
+- Compute Engine (Not strictly necessary. Bots can run anywhere.)
```
Comment (Dor1s) [inline docs/clusterfuzz/architecture.md]: Kevin, do you understand what "bots" mean here? Should we say "Fuzzing bots" to be a bit more explicit?

File: docs/clusterfuzz/architecture.md [hunk 2]
```diff
@@ -21,35 +22,41 @@ triaging crashes, minimizing reproducers, [bisecting], and verification of fixes
...
 - Cloud Datastore
 - Cloud Pub/Sub
 - BigQuery
 - Stackdriver Logging and Monitoring
 
-**Note**: The only bug tracker supported now is the Chromium hosted
+**Note**: The only bug tracker we currently support is the Chromium-hosted
 [Monorail](https://opensource.google.com/projects/monorail). Support for custom
-bug trackers will be added in the near future.
+bug trackers will be added in the future.
```
Comment (Dor1s) [inline docs/clusterfuzz/architecture.md]: (Kevin, please ignore) @oliverchang do you want to re-write this sentence to mention bug tracker extendability?

File: docs/getting-started/prerequisites.md [hunk 3]
```diff
@@ -7,67 +7,67 @@ permalink: /getting-started/prerequisites/
...
 ```
-where X.Y.Z is the release version (e.g. 1.0.1). 
+where X.Y.Z is the release version (for example, 1.0.1). 
 
 ## Installing prerequisites
 
-### Google Cloud SDK
-Install the Google Cloud SDK by following the instructions
-[here](https://cloud.google.com/sdk/).
+### Google Cloud SDK
```
Comment (Dor1s) [inline docs/getting-started/prerequisites.md]: nit: trailing space on this line?

File: docs/clusterfuzz/architecture.md (hunk 1 above)
Comment (kplybon) [reply docs/clusterfuzz/architecture.md]: Good point! I also linked to the section below.

File: docs/getting-started/prerequisites.md (hunk 3 above)
Comment (kplybon) [reply docs/getting-started/prerequisites.md]: Done.

File: docs/clusterfuzz/architecture.md [hunk 4]
```diff
@@ -21,35 +22,41 @@ triaging crashes, minimizing reproducers, [bisecting], and verification of fixes
...
 
 ### Local instance
 It's possible to run ClusterFuzz locally without these dependencies by using
-local Google Cloud emulators, but some features which depend on BigQuery and
+local Google Cloud emulators. If you do, some features that depend on BigQuery and
 Stackdriver will be disabled due to lack of emulator support.
 
+**Note:** Local development is only supported on **Linux** and **macOS**.
```
Comment (jonathanmetzman) [inline docs/clusterfuzz/architecture.md]: maybe "instances are" instead of "development is"

File: docs/getting-started/prerequisites.md [hunk 5]
```diff
@@ -7,67 +7,67 @@ permalink: /getting-started/prerequisites/
...
-purposes.
+Platform](https://cloud.google.com) services, but it's possible to run it locally without these dependencies for testing purposes. See the [Architecture page]({{ site.baseurl }}/architecture/#requirements) for more details.
 
-While ClusterFuzz runs on a number of platforms, local development is only
-supported on **Linux** and **macOS**.
+**Note:** Local development is only supported on **Linux** and **macOS**.
 
 ## Getting the code
+
+Clone the ClusterFuzz repository to your machine by running the following command:
```
Comment (jonathanmetzman) [inline docs/getting-started/prerequisites.md]: meh, I suspect that bash commands are understandable on their own to our target audience

File: docs/clusterfuzz/architecture.md (hunk 4 above)
Comment (kplybon) [reply docs/clusterfuzz/architecture.md]: Done.

File: docs/getting-started/prerequisites.md (hunk 5 above)
Comment (kplybon) [reply docs/getting-started/prerequisites.md]: I'm sure you're right - just for completeness :)

File: docs/clusterfuzz/architecture.md (hunk 2 above)
Comment (oliverchang) [reply docs/clusterfuzz/architecture.md]: Sure, added to my TODO.

**Pull request merged**